Changelog

```diff
@@ -7,6 +7,14 @@
 
 .. _version-3.2.1+edx.1:
```

Comment:
is this line supposed to be updated to `.. _version-3.2.1+edx.2:`?

Reply:
Yes, good catch; fixing.